src/SqlPersistence/Saga/SagaPersister_Get.cs

```diff
@@ -78,8 +78,9 @@ static async Task<Concurrency<TSagaData>> GetSagaData<TSagaData>(SynchronizedSto
             command.Transaction = sqlSession.Transaction;
             var dbCommand = command.InnerCommand;
             appendParameters(dbCommand.CreateParameter, parameter => dbCommand.Parameters.Add(parameter));
+
             // to avoid loading into memory SequentialAccess is required which means each fields needs to be accessed
-            using (var dataReader = await command.ExecuteReaderAsync(CommandBehavior.SingleRow | CommandBehavior.SequentialAccess).ConfigureAwait(false))
+            using (var dataReader = await command.ExecuteReaderAsync(CommandBehavior.SequentialAccess).ConfigureAwait(false))
```

Comment:
@lailabougria I noticed this when I cherry picked the `master` commit. We seem to have removed the `SingleRow` here. Do you remember if that was intentional? I think we might have done this by mistake.

Reply:
You are 100% correct. PR raised to fix master: https://github.com/Particular/NServiceBus.Persistence.Sql/pull/617